### PR TITLE
Switch to differentiable head, larger input sequence

### PR DESCRIPTION
**Pull request by danielkorat**
- Change the model head to SetFitHead t alleviate throughput bottleneck 
 - Add instructions for installing TCMALLOC (required)
 - More robust benchmarking using random sequences of length 256
 
@tomaarsen

### CODE REVIEW

**Review by tomaarsen — APPROVED** [review]
Looks solid as far as I can tell. I can merge this when you're ready.

**Comment by danielkorat** [comment]
Thanks
@Wasserblat, ***@***.***> should we merge or wait for the blog release?

________________________________

**Review by tomaarsen — APPROVED** [review]

**Pull request merged**
